admin_guide/managing_networking.adoc

```diff
@@ -903,3 +903,50 @@ in all other projects. The project to be isolated must first be configured to
 opt in to isolation by setting the proper annotation on its Namespace object,
 and then creating `NetworkPolicy` objects indicating the incoming connections to
 be allowed.
+
+. Create the `NetworkPolicy` objects:
++
+[source,yaml]
+----
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
+metadata:
+  name: allow-http-and-https
+spec:
+podSelector:
+ingress:
+- ports:
+  - protocol: TCP
+    - port: 80
+    - port: 443
+----
++
+Optionally, set the `name` parameter to `default-deny` to ensure that the pods that aren't matched to a `NetworkPolicy` continue to be fully-isolated. Delete any existing `NetworkPolicy` objects that previously had the different value. For example, if you are changing the value from `allow-http-and-https` to `default-deny`, delete the old `NetworkPolicy` objects.
```

Comment:
To clarify, there's nothing magical about the name "default-deny"; if you want to ensure that all pods in a Namespace are isolated, the way to do that is to create a NetworkPolicy that matches every pod (`spec.podSelector` is empty) but doesn't contain any `ingress` entries. You can call the policy whatever you want, "default-deny" is just a suggestion.